server/routes/makeAReferral/makeAReferralController.ts

```diff
@@ -526,6 +528,33 @@ export default class MakeAReferralController {
     ControllerUtils.renderWithLayout(res, view, serviceUser)
   }
 
+  async editOasysRiskInformation(req: Request, res: Response): Promise<void> {
+    if (!config.apis.assessRisksAndNeedsApi.riskSummaryEnabled) {
+      throw createError(403, `access restricted when risk feature flag disabled`, {
+        userMessage: 'You are not authorized to access this page',
+      })
+    }
+    const { accessToken } = res.locals.user.token
+    const referralId = req.params.id
+    const referral = await this.interventionsService.getDraftReferral(accessToken, referralId)
+    const [serviceUser, riskSummary, supplementaryRiskInformation] = await Promise.all([
+      this.communityApiService.getServiceUserByCRN(referral.serviceUser.crn),
+      this.assessRisksAndNeedsService.getRiskSummary(referral.serviceUser.crn, accessToken),
+      this.assessRisksAndNeedsService.getSupplementaryRiskInformationForCrn(referral.serviceUser.crn, accessToken),
+    ])
+
+    if (riskSummary === null) {
+      throw createError(500, `could not obtain risk summary`, {
```

Comment:
When would Risk summary be `null`? Is it an error from the API or when nothing is found? Just wondering if this should return a 404 (or something to that effect) if the risk summary wasn't found for a specific user

Reply:
ah, I messed up here ... I'm displaying empty risk info if it is null now